var/spack/repos/builtin/packages/py-cgal-pybind/package.py

```diff
@@ -10,15 +10,25 @@ class PyCgalPybind(PythonPackage):
     """Internal Python bindings for CGAL"""
 
     homepage = "https://bbpcode.epfl.ch/browse/code/common/cgal-pybind/tree/"
-    git      = "ssh://bbpcode.epfl.ch/common/cgal-pybind"
 
-    version('develop', submodules=True)
-    version('0.0.2', commit='7aa1382d1628ccd51f692750a2b145b1df0694d9', submodules=True)
+    version("develop", submodules=True)
+    version(
+        "0.1.0",
+        tag="cgal_pybind-v0.1.0",
+        submodules=True,
+        git="git@bbpgitlab.epfl.ch:nse/cgal-pybind.git",  # migration to gitlab
+    )
+    version(
```

Comment:
And drop this older version, assuming you did not migrate the tags? If you migrated the tags to GitLab, I don't see a reason to keep referring to Gerrit.

Reply:
@matz-e Thanks for your comments.

The reason why I want to keep the old version 0.0.2 is because another software and its CI depends on it (molecular-systems/skeletonizer). The version 0.0.2 had no tag, hence the commit hash.

In the next commit, I refer only to the gitlab repository.

Reply:
@matz-e I was wrong when I said that py-cgal-pybind is used by the CI of molecular-systems/Skeletonizer.
The CI always builds the develop version. However, the file setup.py of Skeletonizer refers to cgal-pybind>=0.0.1. But this has no immediate consequences on users and deployment.

I have removed version 0.0.2 in the next commit.